<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>510</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>931</width>
    <height>510</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>931</width>
    <height>510</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Lappbill</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="CustomerBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>100</y>
      <width>471</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string>Asiakas</string>
    </property>
    <widget class="QComboBox" name="CustomerList">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>24</y>
       <width>374</width>
       <height>32</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Ei tallennettuja asiakkaita</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>59</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Puh.</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Customer_street1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>138</y>
       <width>371</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Kaupunki</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Customer_Puh">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>371</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Customer_street2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>176</y>
       <width>371</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>64</x>
       <y>176</y>
       <width>0</width>
       <height>0</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Lisäkenttä 1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Customer_Name1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>62</y>
       <width>371</width>
       <height>32</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>59</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Katu1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>59</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Nimi</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Customer_City">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>214</y>
       <width>371</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>59</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Katu2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Customer_Name2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>290</y>
       <width>371</width>
       <height>32</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Vapaaehtoinen</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RemoveCustomerButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>370</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Poista valittu asiakas</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SaveCustomerButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>370</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Tallenna asiakas</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Potilas">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>330</y>
       <width>371</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Vapaaehtoinen</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Lisäkenttä 2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Customer_postal">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>250</y>
       <width>371</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Postinro.</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ProductBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>451</height>
     </rect>
    </property>
    <property name="title">
     <string>Tuote</string>
    </property>
    <widget class="QPushButton" name="Add_Product">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>260</y>
       <width>81</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Lisää</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Product_Name">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>241</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>59</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Nimi</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>59</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Hinta</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>220</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Kpl</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="Product_Price">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
     <property name="maximum">
      <double>99999.990000000005239</double>
     </property>
    </widget>
    <widget class="QSpinBox" name="Product_kpl">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>220</y>
       <width>61</width>
       <height>32</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
    <widget class="QTableWidget" name="OutputProducts">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>311</width>
       <height>251</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>491</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Nimi</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Kpl</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Hinta</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="Remove_Product_from_List">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>410</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Poista valittu</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ProductList">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>241</width>
       <height>32</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Ei tallennettuja tuotteita</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="Save_Product_Button">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Tallenna tuote</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Remove_Product">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>150</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Poista tuote</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>471</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Lasku</string>
    </property>
    <widget class="QDateEdit" name="DueDate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>110</width>
       <height>26</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="currentSectionIndex">
      <number>0</number>
     </property>
     <property name="timeSpec">
      <enum>Qt::LocalTime</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="SaveButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Tallenna PDF</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Eräpäivä</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>146</x>
       <y>30</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Laskunnumero</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="Invoice_Number">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>151</width>
       <height>26</height>
      </rect>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>100.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000000000000000000.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Settings_Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Asetukset</string>
    </property>
   </widget>
  </widget>
  <action name="actionCustomer">
   <property name="text">
    <string>Customer</string>
   </property>
  </action>
  <action name="actionProduct">
   <property name="text">
    <string>Product</string>
   </property>
  </action>
  <action name="actionInvoice">
   <property name="text">
    <string>Invoice</string>
   </property>
  </action>
  <action name="actionNew_customer">
   <property name="text">
    <string>New Customer</string>
   </property>
  </action>
  <action name="actionNew_Product">
   <property name="text">
    <string>New Product</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>ProductList</tabstop>
  <tabstop>Product_Name</tabstop>
  <tabstop>Product_Price</tabstop>
  <tabstop>Save_Product_Button</tabstop>
  <tabstop>Remove_Product</tabstop>
  <tabstop>Product_kpl</tabstop>
  <tabstop>Add_Product</tabstop>
  <tabstop>CustomerList</tabstop>
  <tabstop>Customer_Name1</tabstop>
  <tabstop>Customer_Puh</tabstop>
  <tabstop>Customer_street1</tabstop>
  <tabstop>Customer_street2</tabstop>
  <tabstop>Customer_City</tabstop>
  <tabstop>Customer_postal</tabstop>
  <tabstop>Customer_Name2</tabstop>
  <tabstop>Potilas</tabstop>
  <tabstop>SaveCustomerButton</tabstop>
  <tabstop>RemoveCustomerButton</tabstop>
  <tabstop>SaveButton</tabstop>
  <tabstop>OutputProducts</tabstop>
  <tabstop>Remove_Product_from_List</tabstop>
  <tabstop>DueDate</tabstop>
  <tabstop>Invoice_Number</tabstop>
  <tabstop>Settings_Button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
